### PR TITLE
File Upload and Preview

### PR DESCRIPTION
**Pull request by darshbaxi**
Fixes: #623 
I have made the file upload and preview app in which users can upload files and preview those files. I developed it using the block of million.js.
image
image

**Status**

- [x] Code changes have been tested against prettier, or there are no code changes
- [x] I know how to update typings and have done so, or typings don't need updating

**Semantic versioning classification:**

- [ ] This PR changes the codebase
  - [ ] This PR includes breaking changes (methods removed or renamed, parameters moved or removed)
  - [ ] This PR changes the internal workings with no modifications to the external API (bug fixes, performance improvements)
- [ ] This PR **only** includes non-code changes, like changes to documentation, README, etc.

### CODE REVIEW

File: packages/kitchen-sink/src/examples/file-upload-preview.jsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+import { block } from 'million/react';
+
+export default function File() {
+  const [file, setFile] = useState();
+  function getFile(event) {
+    setFile(URL.createObjectURL(event.target.files[0]));
+  }
+
+  const FileUpload=block(({file})=>{
+    console.log(file)
```
Comment (deeola) [inline packages/kitchen-sink/src/examples/file-upload-preview.jsx:11]: Hey, you left this here😉

File: packages/kitchen-sink/src/examples/file-upload-preview.jsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    return(
+      <div className='preview' >
+      <img src={file} />
```
Comment (deeola) [inline packages/kitchen-sink/src/examples/file-upload-preview.jsx:14]: can we have an 'alt' too?

**Comment by darshbaxi** [comment]
@deeola suggested changes made. please review it again:)

**Comment by aidenybai** [comment]
Please run `pnpm run cleanup`

**Comment by darshbaxi** [comment]
@aidenybai requirement satisfy to run 'pnpm -w run cleanup' command. please review it again:)

**Comment by tobySolutions** [comment]
Have you worked on the requested changes @deeola?

**Comment by darshbaxi** [comment]
@tobySolutions Yes, I made the all requested changes suggested.

**Review by tobySolutions — CHANGES_REQUESTED** [review]
Can you separate the components? It's not really best practice to nest components. Thank you.

**Comment by darshbaxi** [comment]
@tobySolutions Required changes made, please review it again:)

**Comment by darshbaxi** [comment]
@deeola,@tobySolutions I made changes that were suggested by you. please review my PR.

**Review by darshbaxi — COMMENTED** [review]
@tobySolutions,I made the suggested changes:)

**Review by tobySolutions — APPROVED** [review]
LGTM!

**Pull request merged**